### PR TITLE
Allow custom element properties to have associated attributes

### LINKED ISSUE #36: add reflect to property or attribute
There should be a way to signal whether or not a property reflects to it's associated attribute when set

### PR DESCRIPTION
**Pull request by justinfagnani**
Fixes #36

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Added an optional `"syntax"` field to CSSCustomProperty to describe the property syntax using CSS Properties and Values API's syntax strings. Fixes
 https://github.com/webcomponents/custom-elements-manifest/issues/68
 
+- Added CustomElementField that extends ClassField with the abilaity to
```
Comment (thepassle) [inline CHANGELOG.md:28]:
```suggestion
- Added CustomElementField that extends ClassField with the ability to
```

**Comment by thepassle** [comment]
Should we upgrade the `schemaVersion`?

**Comment by justinfagnani** [comment]
> Should we upgrade the schemaVersion?

Yeah, but we have a few changes queued up. We should bump that as part of a release, maybe on Wednesday? (we're off M and T).

**Comment by thepassle** [comment]
Okay sure 👍

This LGTM, ill take a closer look at the interface PR tomorrow

**Pull request merged**